service/hook/bitbucketv2/bitbucketv2_test.go

```diff
@@ -41,6 +41,24 @@ const (
 }
 }`
 
+	sampleMercurialCodePushData = `{
```

Comment:
do a full copy of https://github.com/bitrise-io/bitrise-webhooks/pull/46/files#diff-6cb48c1f65b9a4bc3410e7abd7829c31R15 - your test below expects that ( https://github.com/bitrise-io/bitrise-webhooks/pull/46/files#diff-6cb48c1f65b9a4bc3410e7abd7829c31R803 )